client/lcd/root.go

```diff
@@ -82,36 +82,30 @@ func (rs *RestServer) Start(listenAddr string, sslHosts string,
 	})
 
 	// TODO: re-enable insecure mode once #2715 has been addressed
```

Comment:
I think we can remove this now, yeah?

Reply:
My change was lost in a rebase. Amended now, thanks.